ci/rpm_build_unsuccessful.sh

```diff
@@ -0,0 +1,24 @@
+# /bin/sh
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]